interface/drugs/dispense_drug.php

```diff
@@ -46,6 +46,7 @@ function send_email($subject, $body)
 $quantity        = $_REQUEST['quantity'];
 $fee             = $_REQUEST['fee'];
 $user            = $_SESSION['authUser'];
+$encounter            = $_SESSION['encounter'];
```

Comment:
(this is my fault since I gave you wrong command during the conference call. Should use this command instead of above):
$encounter = $_SESSION['encounter'] ?? 0;